packages/node/src/evm/verification/request-verification.test.ts

```diff
@@ -41,19 +41,6 @@ describe('verifyDesignatedWallets', () => {
     }
   });
 
-  it('ignores API calls that have no requester index', () => {
```

Comment:
This is duplicated (same test name with same code) so I removed one copy.
@andreogle I see many tests with the same name in code, maybe there was some bad rebase along the way?

Reply:
Could be yeah, or maybe just bad-copy pasting

Reply:
I'll check that in another PR

Reply:
Actually, this is my bad - this test is NOT duplicated and shouldn't be removed. I'll revert back.